website/docs/reference/sdks/index.md

```diff
@@ -103,6 +90,102 @@ If you see an item marked with a ❌ that you would find useful, feel free to re
 | Bootstrap from file | ✅ | ✅ | ✅ | ⭕ | ✅ | ✅ | ✅ | ⭕ |
 | Custom Bootstrap implementation | ✅ | ✅ | ✅ | ⭕ | ✅ | ✅ | ✅ | ⭕ |
 
+
+### Client-side SDKs
+
+Client-side SDKs can connect to [Unleash Edge](../../generated/unleash-edge.md), the [Unleash front-end API](../front-end-api.md), or to the [Unleash Proxy](../../generated/unleash-proxy.md), but _not_ to the regular Unleash client API.
+
+The list of official client-side SDKs is:
+
+- [Android SDK](/docs/generated/sdks/client-side/android-proxy.md)
+- [Flutter Proxy SDK](/docs/generated/sdks/client-side/flutter.md)
+- [iOS Proxy SDK](/docs/generated/sdks/client-side/ios-proxy.md)
+- [Javascript SDK](/docs/generated/sdks/client-side/javascript-browser.md)
+- [React Proxy SDK](/docs/generated/sdks/client-side/react.md)
+- [Svelte Proxy SDK](/docs/generated/sdks/client-side/svelte.md)
+- [Vue Proxy SDK](/docs/generated/sdks/client-side/vue.md)
+
+Additionally, the [Next.js SDK](/docs/generated/sdks/client-side/next-js.md) also supports a client-side mode.
+
+
+#### Client-side SDK compatibility tables
+
+Because client-side SDKs don't evaluate their features locally, some of their capabilities depend on what they are connected to.
+
+One clear difference between server-side SDKs and client-side SDKs, is that client-side SDKs don't support using fallback values in `isEnabled` calls. This is because the evaluation happens upstream from them.
+
+| Capability | [Android](/docs/generated/sdks/client-side/android-proxy.md) | [Flutter](/docs/generated/sdks/client-side/flutter.md) | [iOS](/docs/generated/sdks/client-side/ios-proxy.md) | [JavaScript](/docs/generated/sdks/client-side/javascript-browser.md) | [React](/docs/generated/sdks/client-side/react.md) | [Svelte](/docs/generated/sdks/client-side/svelte.md) | [Vue](/docs/generated/sdks/client-side/vue.md) |
+| --- | :-: | :-: | :-: | :-: | :-: | :-: | :-: |
+| **Category: Initialization** |  |  |  |  |  |  |  |
+| Async initialization | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ❌ |
+| Can block until synchronized | ✅ | ✅ | ✅ | ⭕ | ⭕ | ✅ | ✅ |
+| Default refresh interval | 10s | 15s | 15s | 15s | 15s | 30s | 30s |
```

Comment:
E.g for Flutter it's 30 seconds for the refresh interval and the metrics interval. But maybe instead of syncing the table we should sync the SDKs?

Reply:
That we could, but it might also make sense for different SDKs to have different intervals. I'd be all for synchronizing the defaults, but I think we can do that as a separate thing.